pylint/checkers/refactoring.py

```diff
@@ -102,6 +103,11 @@ class RefactoringChecker(checkers.BaseTokenChecker):
                   'weird bugs in your code. You should always use parentheses '
                   'explicitly for creating a tuple.',
                   {'minversion': (3, 0)}),
+        'R1708': ('Sub package folder (%s) conflict with sub package python '
```

Comment:
Message seems to be left-over from previous revision. We decided to drop sub- prefixes.